08_files_lab/02.py

```diff
@@ -9,22 +9,15 @@
 import argparse
 import itertools
 
-def is_valid_file_path(file_path):
-    if os.path.isfile(file_path):
-        return True
-    else :      
-        print "Error - invalid file path",file_path
-        return False
-
 def create_dictionaries(file_path):
-    if not os.path.exists(os.path.dirname(file_path)):
-        try:
-            os.makedirs(os.path.dirname(file_path))
-        except OSError as exc: 
-            if exc.errno != errno.EEXIST:           
-                return False
-    return True
+    dir = os.path.dirname(file_path)
+    if not os.path.exists(dir):
+        os.makedirs(os.path.dirname(file_path))

```

Comment:
1. You mean create_directories right?
2. P.S. one of my favourite python3 features is:
https://docs.python.org/3/library/os.html?highlight=makedirs#os.makedirs